src/Core/Services/ExportService.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Bit.Core.Abstractions;
+using Bit.Core.Enums;
+using Bit.Core.Models.Export;
+using Bit.Core.Models.View;
+using Bit.Core.Utilities;
+using CsvHelper;
+using CsvHelper.Configuration;
+using CsvHelper.Configuration.Attributes;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Bit.Core.Services
+{
+    public class ExportService : IExportService
+    {
+        private readonly IFolderService _folderService;
+        private readonly ICipherService _cipherService;
+
+        private List<FolderView> _decryptedFolders;
+        private List<CipherView> _decryptedCiphers;
+
+        public ExportService(
+            IFolderService folderService,
+            ICipherService cipherService)
+        {
+            _folderService = folderService;
+            _cipherService = cipherService;
+        }
+
+        public async Task<string> GetExport(string format = "csv")
+        {
+            _decryptedFolders = await _folderService.GetAllDecryptedAsync();
+            _decryptedCiphers = await _cipherService.GetAllDecryptedAsync();
+
+            if(format == "csv")
+            {
+                var foldersMap = _decryptedFolders.Where(f => f.Id != null).ToDictionary(f => f.Id);
+
+                var exportCiphers = new List<ExportCipher>();
+                foreach(var c in _decryptedCiphers)
+                {
+                    // only export logins and secure notes
+                    if(c.Type != CipherType.Login && c.Type != CipherType.SecureNote)
+                    {
+                        continue;
+                    }
+
+                    if(c.OrganizationId != null)
+                    {
+                        continue;
+                    }
+
+                    var cipher = new ExportCipher();
+                    cipher.Folder = c.FolderId != null && foldersMap.ContainsKey(c.FolderId)
+                        ? foldersMap[c.FolderId].Name
+                        : null;
```

Comment:
I would move the else condition up 1 line, but that's just my formatting preference